src/frequenz/sdk/config/_config_managing.py

```diff
@@ -130,6 +163,12 @@ def _read_config(self) -> abc.Mapping[str, Any]:
             except ValueError as err:
                 _logger.error("%s: Can't read config file, err: %s", self, err)
                 error_count += 1
+            except FileNotFoundError:
+                # It is ok for config file to don't exist.
+                _logger.info(
+                    "%s: Config file %s not found. Ignoring it.", self, config_path
+                )
+                error_count += 1
```

Comment:
So I log info, if any config file is doesn't exist. 
And (code later) raises ValueError if no config file exist.

Reply:
> Luca, this PR was to fix existing bugs, not redesign ConfigManger. I guess it needs redesign and rethinking but maybe in separate PR and after we agree how it should work.

Yes, 100% agreed, this is also why I said re-evaluating what to do on a failure to read a config file is for another PR.

But catching other types of errors is still a bug that IMHO is in scope for this PR. Since you are catching a new `FileNotFound` why not just catching all `OSError`s?

> So here I log info, if any config file is doesn't exist.
> And (code 3 lines below) raises ValueError if no config file exist. (at least one must exist)

Yeah, but that's only if we can't read **any** config file, if only one failed, no `ValueError` will be raised, and the missing file won't be noticed if we are only monitoring WARNING and up. Also it is more consistent to how we handle the TOML parsing `ValueError`. We log an error if TOML parsing fails. I don't understand why the inconsistency of making one issue emit an error log and the new one just a info.

Reply:
* https://github.com/frequenz-floss/frequenz-sdk-python/issues/1120

Reply:
Ok the whole discussion was because I had impression that having not all config files is not an error.
But it looks like it is unexpected so as you suggested I changed severity of this logging to  error. :) 
Sorry for my missunderstanding

Reply:
No it is OK, originally it was thought like that, only one config file should be enough, but I think that's assuming too much, and is hard to know if missing files are errors or expected, so I opened the new issue to change that in the future.

Reply:
Oh, I see in the last commit you are changing this. I would not address the new issue (#1120) in this PR, I think it needs a bit more thought and it is not strictly related to the bug fixes in this PR.